java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -58,8 +58,8 @@
   private final BufferManager bufManager;
   private final ArrowByteBufAllocator alloc;
   private final boolean isEmpty;
-  private final HistoricalLog historicalLog = BaseAllocator.DEBUG ?
-      new HistoricalLog(BaseAllocator.DEBUG_LOG_LENGTH, "ArrowBuf[%d]", id) : null;
+  private final HistoricalLog historicalLog = BaseAllocator.DEBUG
+      ? new HistoricalLog(BaseAllocator.DEBUG_LOG_LENGTH, "ArrowBuf[%d]", id) : null;
```

Comment:
Does the style require "?" to be on the second line?

Reply:
yup, see the above comment